.gitignore

```diff
@@ -21,7 +21,11 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+<<<<<<< Updated upstream
```

Comment:
There seems to be an unresolved merge error, can you try rebasing your branch on main and fixing this conflict?